### DISCUSSION TITLE
How to convert "{% assign items = Queries[query] | query: size: fetchCount, from: from %}" to razor code,please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#16689] [Q&A] [answered]
How to convert the following liquid code to razor code,please?
{% assign query = Model.ContentItem.Content.ContentFeed.Items.Value %}
{% assign items = Queries[query] | query: size: fetchCount, from: from %}

**Comment by Habbni** [comment, marked as answer]
```
@{
    var items = (await Orchard.ContentQueryAsync(query)).Skip(from-1).Take(size);
}
```
hope i could help.
it´s also quite well documented here: https://docs.orchardcore.net/en/latest/reference/modules/Queries/#razor-helpers
